src/test/java/com/butbetter/applicationservices/Faker/WrongProduct.java

```diff
@@ -0,0 +1,70 @@
+package com.butbetter.applicationservices.Faker;
+
+import com.butbetter.applicationservices.db.model.Product;
+import com.butbetter.applicationservices.db.model.ProductBeverageEnum;
+import com.butbetter.applicationservices.db.model.ProductRatingEnum;
+
+import java.math.BigDecimal;
+
+public class WrongProduct {
```

Comment:
Du hast die andere klasse "ProductFaker" genannt
am besten hier auch Faker hinten dran setzen oder vielleicht sogar beides in eine klasse?

Reply:
Alles klar